src/battle_ai_util.c

```diff
@@ -4517,3 +4575,75 @@ bool32 HasBattlerSideAbility(u32 battler, u32 ability, struct AiLogicData *aiDat
         return TRUE;
     return FALSE;
 }
+
+u32 FriendlyFireHitsToKO(u32 battler)
+{
+    if (AI_THINKING_STRUCT->aiFlags[battler] & AI_FLAG_RISKY)
+        return FRIENDLY_FIRE_RISKY_THRESHOLD;
+    if (AI_THINKING_STRUCT->aiFlags[battler] & AI_FLAG_CONSERVATIVE)
+        return FRIENDLY_FIRE_CONSERVATIVE_THRESHOLD;
+
+    return FRIENDLY_FIRE_NORMAL_THRESHOLD;
+}
+
+bool32 IsMoxieTypeAbility(u32 abilityId)
+{
+    switch (abilityId)
```

Comment:
```suggestion
bool32 IsMoxieTypeAbility(u32 ability)
{
    switch (ability)
```
for consistency :sweat_smile: